tests/performance/apdex-score/test/page-objects/login.page.js

```diff
@@ -3,23 +3,23 @@ const Page = require('./page');
 
 class LoginPage extends Page {
   
-  get inputUsername () {
+  get inputUsername() {
     return $('//android.view.View[1]/android.widget.EditText');
   }
 
-  get inputPassword () {
+  get inputPassword() {
     return $('//android.view.View[3]/android.widget.EditText');
   }
 
-  get btnLogin () {
+  get btnLogin() {
     return $('//*[@text="Login"]');
   }
 
-  get btnAccept () {
+  get btnAccept() {
     return $('//*[@text="Accept"]');
   }
```

Comment:
Same as the previous comment, maybe we can have a common `get button` method in `page.js` and you only need to send the text of the button that you want to access as a parameter.

Reply:
@ralfudx this one is fast, you do it like this: 

```
get button(label) {
    return $('//*[@text="' + label +'"]');
}
```

Then you use it like this:

```
loginPage.button('Login');
loginPage.button('Accept'); 
```

Reply:
`A 'get' accessor cannot have parameters.ts(1054)`